packages/calypso-products/src/main.ts

```diff
@@ -718,7 +718,7 @@ export const getPopularPlanSpec = ( {
 	isJetpack,
 	availablePlans,
 }: {
-	flowName: string;
+	flowName?: string | null;
```

Comment:
The typing here says that flowName may be `undefined`, `string`, or `null` Just wanna confirm that this is the intention?

Reply:
Yeah, intentional. A lot of these types are created to match underlying selectors that may return either or